.github/ISSUE_TEMPLATE/fix-suggestions.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Fix Suggestions
+about: Add a new component with fix suggestions to existing check
+title: "[Fix Suggestions]"
+labels: autofix
+assignees: ''
+
+---
+
+**Which Atlas check is the fix suggestion related to?** 
+
+Name of Atlas check that need to be added the suggested fix component. Link to associate check script
+
+
+**Describe more details of suggested fix(es)**
+
+*Suggested Fix 1*: 
+
+- What type? (tag, geometry). 
+
+- What operation is needed? (add, modify, delete)
+
+- What detailed action is of the suggested fix? (A clear and concise description of what the suggested fix does. Ex. Remove tag "area=yes" when [...])
+
+*Suggested Fix 2 (if any)*:
+
+If there are multiple suggested fixes related to the same Atlas check required, please add another section with the details above
```

Comment:
nit: If there are multiple suggested fixes related to the same Atlas Check ~required~, please add another section with the _above_ details